lib/src/models/calendar_date_picker2_config.dart

```diff
@@ -181,6 +182,9 @@ class CalendarDatePicker2Config {
   /// Only applicable when [calendarType] is [CalendarDatePicker2Type.range].
   final bool rangeBidirectional;
 
+  // Choose calendar scroll physics
```

Comment:
`//` is a regular comment while `///` is for documentation
```suggestion
  /// Choose calendar scroll physics
```